books/RayTracingTheRestOfYourLife.html

```diff
@@ -348,7 +348,7 @@
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
             sqrt_spp = int(sqrt(samples_per_pixel));
-            pixel_sample_scale = 1.0 / (sqrt_spp * sqrt_spp);
+            pixel_samples_scale = 1.0 / (sqrt_spp * sqrt_spp);
```

Comment:
A part of me wants to rewrite this line as `pixel_samples_scale = recip_sqrt_spp * recip_sqrt_spp;` but I'm not sure if the opaqueness is worth it.

Reply:
Heh, I had the same thought. It was the opaqueness that made me decide against it. I'll keep it as is.